backend/src/main/resources/graphql/main.graphqls

```diff
@@ -310,6 +310,15 @@ type UploadResult {
   errors: [FeedbackMessage]
 }
 
+type UploadResponse {
```

Comment:
This is very slightly distinct from the `UploadResult` above, which is an upload record from the SimpleReport database. This type represents a response from ReportStream's submissions endpoints. The biggest difference is that there will not be an `internalId` returned. If this feels confusing to anyone we could probably work to consolidate these types and make that property optional.

Reply:
yeah, I'd be in favor of either consolidating the types or at least renaming this one to be more clear

Reply:
This makes a _lot_ of sense to do but is a little trickier than I thought. Significantly, `created_at` is a private and immutable property on a `TestResultUpload` and is generated on-save. We can't manually set that property to the timestamp of the submission. I think I will keep returning `UploadResponse` and mull on how to simplify these types going forward.